packages/toolkit/connector-kit/src/types.ts

```diff
@@ -112,6 +123,8 @@ export type BaseConnector<Type extends ConnectorType> = {
 
 export type CreateConnector<T extends BaseConnector<ConnectorType>> = (options: {
   getConfig: GetConnectorConfig;
+  getStorage?: GetStorage;
```

Comment:
`getSession`? Because "storage" sounds like persist storage, which we may use it in the future.